### ISSUE TITLE
Messages: use aria-label when href is empty

### ISSUE THREAD

**Issue by Eehol** [primefaces/primefaces#4824]
## 1) Environment
- PrimeFaces version: 7.0.2 (primefaces showcase)
- Does it work on the newest released PrimeFaces version? Version? No
- Affected browsers: All

## 2) Expected behavior

The messages that can be closed have a cross at the right of the red/yellow/.. box containing the message and it should be accessible.

Examples : 
- https://www.primefaces.org/showcase/ui/message/messages.xhtml -> click "Info"

- https://www.primefaces.org/showcase/ui/file/upload/single.xhtml -> upload a file with invalid extension (e.g. a .txt file)

The cross is implemented in this way :
`<a href="#" class="ui-messages-close" onclick="..."><span class="ui-icon ui-icon-close"></span></a>`

The link is not accessible because it is empty (href="#") and it has no aria-label.

Solution : either add a generic aria-label[ as you did for the cross of the p:dialog](https://www.primefaces.org/showcase/ui/overlay/dialog/basic.xhtml) component (aria-label="Close"), or provide an attribute to give a customized aria-label.

NB: The aria-label="Close" solution work but it is not possible to i18n !

**Comment by melloware** [comment]
@Eehol You can il8n the Close icon of the dialog.  It is in /org/primefaces/Messages.properties.

https://github.com/primefaces/primefaces/blob/master/src/main/resources/org/primefaces/Messages.properties#L15

So in your project if you have a /org/primefaces/Messages_de.properties it should work if your locale is Germany it pick it up and replace the labels.
